index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="navbar">
+      <div id="bg-image-1"></div>
+      <nav class="menu-btn">
+        <img src="src/ic_menu.png" alt="menu" />
+      </nav>
+    </header>
+    <main>
+      <section>
+        <h1>Stepan Leonov</h1>
+        <p class="bio">
+          Hello! I am a software developer! I can help you build a product, a
+          feature or a website. Take a look at my works. If you like what you
+          see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <button type="button" class="contact-button">
+          Start collaboration
+        </button>
+      </section>
+      <div id="bg-image-2"></div>
+    </main>
```

Comment:
- [ ] [REQUIRED] Please apply the necessary margin in between each section so that your design can look just like the template provided. Increase the font size, increase the button size. it is required that you stick to the provided template design as much as possible.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:

![2](https://user-images.githubusercontent.com/58692590/154485237-0296ceba-d80b-448d-b5ec-cd36651d4be9.png) | ![22](https://user-images.githubusercontent.com/58692590/154485256-48b3ac85-9f4b-4c5f-b3fe-a64759c4c226.png)